manim/utils/bezier.py

```diff
@@ -119,6 +120,56 @@ def curve(t):
     return [h0, h1, h2]
 
 
+def split_quadratic_bezier(points: Iterable[float], t: float) -> np.ndarray:
+    """
+    Split a quadratic bezier curve into two quadratic beziers at t
+
+    Parameters
+    ----------
+    points : np.array
+        The control points of the bezier curve
+        has shape [a1, h1, b1]
+
+    t : float
+        The t-value at which to split the bezier curve
+
+    Returns
+    -------
+        returns the two beziers as a list of tuples
+        has the shape [a1, h1, b1], [a2, h2, b2]
+    """
+    a1, h1, a2 = points
+    s1 = interpolate(a1, h1, t)
+    s2 = interpolate(h1, a2, t)
+    p = interpolate(s1, s2, t)
+
+    return np.array([a1, s1, p, p, s2, a2])
+
+
+def subdivide_quadratic_bezier(points: Iterable[float], n: int) -> np.ndarray:
+    """Subdivide a quadratic bezier n times
```

Comment:
A bit more details make sense I think.

```suggestion
    """Subdivide a quadratic Bézier curve ``n`` times.
    
    The points at which the curve is split are located at the
    arguments :math:`t = i/n` for :math:`i = 1, ..., n-1`. 
```